content/guides/references/migration-guide.md

```diff
@@ -470,14 +470,14 @@ module.exports = (on, config) => {
 To run your component tests you _must_ use the dedicated component testing
 subcommands.
 
-- `cypress open-ct`
-- `cypress run-ct`
+- `cypress open --component`
```

Comment:
I don't think we want to change usage instructions for older versions of Cypress, since those version won't have the new functionality. However, it may be valuable for there to be a "Migrating to Cypress ___" section added to the top of the migration guide that explains these changes. Would that be done as part of this PR, or as part of some separate process?